A3-channel-tracing.md

```diff
@@ -50,10 +50,15 @@ We define a minimal set of events that must be traced, in order to consider the
 All of these events must be traced:
 - Channel creation and deletion
 - Subchannel creation and deletion
-- Address resolution resulting from service config change
-- Address resolution that causes number of backends to go from zero to non-zero
 - Channel connectivity state changes
 - Subchannel connectivity state changes
+- Interesting address resolution events (see below for details)
+
+Address resolution is special case. We want to track resolution events, but we do not want to flood the trace buffer with them. So we define several types of "interesting" resolution events that must be traced:
```

Comment:
We should explain why flooding the trace buffer is a concern.  Users who are used to DNS-based environments where backends have fairly static addresses may not understand this.  We need to explain that there are environments where backends are more dynamically scheduled and the resolver will be delivering updates very frequently, in which case we don't want to log every result returned by the resolver, since most of them are far less interesting than the other events that would be pushed out of the trace buffer.

Reply:
Done